app/modules/encounters/resources.py

```diff
@@ -176,19 +96,61 @@ def get(self, encounter):
     )
     @api.login_required(oauth_scopes=['encounters:write'])
     @api.parameters(parameters.PatchEncounterDetailsParameters())
-    @api.response(schemas.DetailedEncounterSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def patch(self, args, encounter):
         """
         Patch Encounter details by ID.
         """
+
+        edm_args = [
+            arg
+            for arg in args
+            if arg.get('path')
+            in parameters.PatchEncounterDetailsParameters.PATH_CHOICES_EDM
+        ]
+        if edm_args and len(edm_args) != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(400, 'Cannot mix EDM patch paths and houston patch paths')
+
+        if edm_args:
+            log.debug(f'wanting to do edm patch on args={args}')
+            try:
+                result_data = current_app.edm.request_passthrough_result(
+                    'encounter.data',
+                    'patch',
+                    {'data': args},
+                    encounter.guid,
+                    request_headers=request.headers,
+                )
+            except HoustonException as ex:
+                log.warning(f'EDM patch got {ex}')
```

Comment:
In theory this shouldn't be necessary as the HoustonException constructor should print everything needed.